### PR TITLE
Landing page styling for claim and new gift

### PR DESCRIPTION
**Pull request by erler**
Styling for the landing pages Claim Gift and New Gift. When the flow is started (`step > 0`) switch to card.

Screenshot 2021-08-19 at 11 09 27
Screenshot 2021-08-18 at 17 21 57

### CODE REVIEW

File: src/kusama-scss/_bg.scss [hunk 1]
```diff
@@ -1,4 +1,12 @@
+/* Standard Background */
+
 body {
+    background-color: #000;
+}
+
+/* Campaign Background */
```
Comment (erler) [inline src/kusama-scss/_bg.scss:7]: For now, just commented the campaign related styling out.

File: src/kusama-scss/_card.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .card {
-    background-color: #000;
```
Comment (erler) [inline src/kusama-scss/_card.scss:2]: This styling was also campaign related

**Comment by hamidra** [comment]
For Kusama, I was wondering if we need to need to add a bit contrast to the card border colors. They seem to be fading in the black background. 
Screen Shot 2021-08-19 at 5 36 58 PM

**Review by hamidra — APPROVED** [review]

**Comment by erler** [comment]
> For Kusama, I was wondering if we need to need to add a bit contrast to the card border colors. They seem to be fading in the black background. May be set it the same as the dropdown border color which is a bit lighter, Though seems the Kusama theme in general is dark so it might be intended, not sure.

Thanks for the feedback. The contrast between border and background is pretty clear on my external screen, but I'll play around with different greys and screens. All greys should be in the same hue for sure.

**Pull request merged**